common/lib/xmodule/xmodule/templates/problem/jsinput_response.yaml

```diff
@@ -5,67 +5,176 @@ metadata:
-          </p>
-          <p>
-              For more information, see
-              <a href="http://edx.readthedocs.io/projects/edx-partner-course-staff/en/latest/exercises_tools/custom_javascript.html" target="_blank">
-              Custom JavaScript Problem</a> in <i>Building and Running an edX Course</i>.
-          </p>
-          <p>
-              JavaScript developers can also see
-              <a href="http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/extending_platform/javascript.html" target="_blank">
-              Custom JavaScript Applications</a> in the <i>EdX Developer's Guide</i>.
-          </p>
-          <p>
-              When you add the problem, be sure to select <strong>Settings</strong>
-              to specify a <strong>Display Name</strong> and other values that apply.
-          </p>
-          <p>You can use the following example problem as a model.</p>
-
-          <customresponse cfn="vglcfn">
-      <script type="loncapa/python">
-      <![CDATA[
+        <script type="loncapa/python">
+
+
       import json
-      def vglcfn(e, ans):
-        '''
-        par is a dictionary that contains two keys, "answer" and "state".
-        The value of "answer" is the JSON string that "getGrade" returns.
-        The value of "state" is the JSON string that "getState" returns.
-        Clicking either "Check" or "Save" registers the current state.
-
-        '''
-        par = json.loads(ans)
-        # You can use the value of the answer key to grade:
-        answer = json.loads(par["answer"])
-        return answer["cylinder"] and not answer["cube"]
-        '''
-        # Or you can use the value of the state key to grade:
-        state = json.loads(par["state"])
-        selectedObjects = state["selectedObjects"]
-        return selectedObjects["cylinder"] and not selectedObjects["cube"]
-        '''
-      ]]>
+
+      ########## Course Author Section #########
+      ## Change whatever you need to in here. ##
+      ##########################################
+
+      # These are REQUIRED for all problems.
+      problem_type = 'interval'  # Set to 'interval' or a 'number'
+      show_open_close = False    # Do we ask for open/closed interval?
+      feedback = True            # Do we tell people how close they were?
+      is_time_question = True    # Should we convert these numbers to times (min:sec)?
+                                 # If this is set True, course authors still use seconds.
+
+      # Only for number-guessing:
+      correct_number = 10            # The right answer
+      tolerance = [2, 4, 6]          # How close do they have to be? [correct, near, far]
+      brackets = [1, 0.5, 0.25, 0]   # Scores for various range brackets
+
+      # Only for interval-guessing:
+      correct_interval = [416,660]               # Please put the lower value on the left.
+      interval_type = ['closed', 'closed']     # Are endpoints 'open' or 'closed'?
```

Comment:
Nit: fix the indenting of the comments, as it appears there was an effort to align all in-line comments.